src/stores/user/models/RandomUserResponseModel.ts

```diff
@@ -0,0 +1,53 @@
+import {BaseModel} from 'sjs-base-model';
+import UserModel from './UserModel';
+
+/*
+    // Returned Api Data Sample
+    {
+      "results": [
+        {
+          "name": {
+            "title": "mrs",
+            "first": "frances",
+            "last": "reynolds"
+          },
+          "email": "frances.reynolds@example.com",
+          "dob": {
+            "date": "1966-03-19T23:24:27Z",
+            "age": 52
+          },
+          "phone": "01-1488-2236",
+          "id": {
+            "name": "TFN",
+            "value": "406524662"
+          },
+          "picture": {
+            "large": "https://randomuser.me/api/portraits/women/50.jpg",
+            "medium": "https://randomuser.me/api/portraits/med/women/50.jpg",
+            "thumbnail": "https://randomuser.me/api/portraits/thumb/women/50.jpg"
+          }
+        }
+      ],
+      "info": {
+        "seed": "cef175b8463ecc54",
+        "results": 1,
+        "page": 1,
+        "version": "1.2"
+      }
+    }
+ */
+export default class RandomUserResponseModel extends BaseModel {
+
+    public readonly results: UserModel[] = [UserModel as any];
```

Comment:
these seems a little odd.  is this because `data` is from an outside source?  at the very least this line deserves some sort of comment on _why_ it is written this way

Reply:
If you look at the `json` data you will see an array of `results`.